torch/csrc/distributed/c10d/logger.cpp

```diff
@@ -192,6 +192,17 @@ void Logger::set_construction_data_and_log(
   at::LogPyTorchDDPUsage(*ddp_logging_data_);
 }
 
+void Logger::set_event_time(
+    int64_t& event_time,
+    Timer& timer,
+    Timer::Event event) {
+  auto timestamp = timer.getTimestamp(event);
+  if (timestamp != c10::nullopt) {
+    // TODO: should we set this as human-readable time instead of unixtime?
```

Comment:
It seems that what are mainly used are the results returned by `measureDifference`, so probably fine.